### ISSUE TITLE
Add the hotkey for switching into the fullscreen mode to the tooltip of the fullscreen button

### ISSUE THREAD

**Issue by rpkoller** [ckeditor/ckeditor5#19783]
## 📝 Provide a description of the new feature

_What is the expected behavior of the proposed feature?_

Noticed while reviewing the issue introducing the full screen functionality in ckeditor 5 for drupal (https://www.drupal.org/project/drupal/issues/3331158). It is related to https://github.com/ckeditor/ckeditor5/issues/19782. 

At the moment the tooltip for the fullscreen functionality looks the following way: 

![Image](https://github.com/user-attachments/assets/058c6009-12b8-49f1-a7a9-9f57fdd05ea3)

unlike other buttons that provide a reminder about the associated hotkey like the bold button the full screen button does not provide that reminder. A user has to leave the context of the editor and search (and in case they find also remember) the hotkey on for example https://ckeditor.com/docs/ckeditor5/latest/features/fullscreen.html?docId=eaee617be2f6646f989ff2b230357ea47

I would suggest to add the hotkey to the tooltip as well (plus fixing the aforementioned issue in https://github.com/ckeditor/ckeditor5/issues/19782) 

---

If you'd like to see this feature implemented, add a 👍 reaction to this post.

**Comment by Witoso** [comment]
Thanks for the report! We would need to check more actions that may be missing it in the editor, @dagdzi @dwandeljanicka.

> A user has to leave the context of the editor and search (and in case they find also remember) the hotkey on for example

Just a note that it's untrue [if accessibility helper is enabled](https://ckeditor.com/docs/ckeditor5/latest/features/accessibility.html#displaying-keyboard-shortcuts-in-the-editor). All shortcuts are then in the editor's helper.

**Comment by rpkoller** [comment]
oh i wasn't aware the accessibility helper plugin existed. on the demo it was/is not enabled and it is also not available in the context of ckeditor in drupal out of the box. thank you for the heads up! that is a good pointer for drupal to provide some guidance in context
